src/file-upload/__tests__/file-upload.test.tsx

```diff
@@ -0,0 +1,363 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Looks like you have some unexpected diff in here. I think that will be fixed if you set main as the merge target.

Reply:
Yes, I made a Git mess by merging main locally; for some reason the merge was made without merge commit and all the merged commits from main appeared as belonging to the PR. Fixed now.